source/dpp/runtime/options.d

```diff
@@ -46,23 +47,15 @@ struct Options {
         else
             enforce(args.length >= 2, "Not enough arguments\n" ~ usage);
 
+        args = response_expand(args);
+
         dppFileNames = args.filter!(a => a.extension == ".dpp").array;
-        enforce(dppFileNames.length != 0, "No .dpp input file specified\n" ~ usage);
 
         // Remove the name of this binary and the name of the .dpp input file from args
         // so that a D compiler can use the remaining entries.
-        dlangCompilerArgs =
-            args[1..$].filter!(a => a.extension != ".dpp").array ~
-            dFileNames;
-
-        // if no -of option is given, default to the name of the .dpp file
-        if(!dlangCompilerArgs.canFind!(a => a.startsWith("-of")))
-            dlangCompilerArgs ~= "-of" ~
-                args.
-                filter!(a => a.extension == ".dpp" || a.extension == ".d")
-                .front
-                .stripExtension
-                ~ exeExtension;
```

Comment:
There's no need to handle this here, if we don't mess with file ordering then we get `dmd`'s behaviour for free.